model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -76,8 +76,8 @@ object AttributeName {
 case class WorkspaceRequest (
                               namespace: String,
                               name: String,
-                              authorizationDomain: Set[ManagedGroupRef],
-                              attributes: AttributeMap
+                              attributes: AttributeMap,
+                              authorizationDomain: Option[Set[ManagedGroupRef]] = Option(Set.empty)
```

Comment:
is None functionally any different that Option(Set.empty)?

Reply:
I guess what I am really asking is should we treat Option(Set.empty) as an error condition, i.e. if the user specifies the field they probably intended to put something in there so we should tell them they didn't?

Reply:
For a person, perhaps. For a program, I'm inclined to allow an empty set. Also, rejecting an empty list would now be a breaking API change.

Reply:
So if None means no auth domain, what would an empty set be?  Hopefully not a workspace nobody is authorized to access.  If that _is_ the case though I'd rather not let people make a workspace unaccessible like that.  If Option(Set.empty) is the same as None that's perhaps confusing, so I'd probably lean toward treating Option(Set.empty) as an error along the lines of "If an authorization domain is defined it needs to have at least one member"

Reply:
@abaumann as this is implemented, None and Set.empty both mean that no authorization domain is set, so you couldn't create an unaccessible workspace. 

I think I agree that for human users of the endpoint, erroring on an empty set is probably the more clear thing to do- but @breilly2 is right- doing it that way would be a new breaking API change since people have updated their scripts to feed in an empty set

Reply:
yeah, unfortunately - so I guess having them equal seems OK, just odd - we should document this well